apps/redi-talent-pool/src/components/organisms/JobListingCard.tsx

```diff
@@ -30,10 +31,10 @@ export function JobListingCard({
   const companyAvatarImage =
     jobListing?.tpCompanyProfile?.profileAvatarImageS3Key
 
-  // const handleFavorite = (e: React.MouseEvent) => {
-  //   e.stopPropagation()
-  //   toggleFavorite && toggleFavorite(profile.id)
-  // }
+  const handleFavorite = (e: React.MouseEvent) => {
```

Comment:
Can we suffix this function name with `Click`?

Reply:
Done!